docs/docs/chitchat-faqs.mdx

```diff
@@ -265,76 +85,112 @@ pipeline:
     epochs: 100
 ```
 
-Now that we've defined the NLU side, we need to make Core aware of these changes. Open your `domain.yml` file and add the `faq` intent:
-
-```yaml
-intents:
-  - greet
-  - bye
-  - thank
-  - faq
-```
-
-We'll also need to add a [retrieval action](./retrieval-actions.mdx),
-which takes care of sending the response predicted from the ResponseSelector back to the user,
-to the list of actions. These actions always have to start with the `respond_` prefix:
+By default, the `ResponseSelector` will build a single retrieval model for all retrieval intents.
```

Comment:
When might you want to build separate models for different intents vs a single model for all retrieval intents?

Reply:
I'm actually second guessing myself on this now - @akelad what would you say?

Reply:
Given that we train on intent labels now it prob makes sense to just do separate models each time. but @dakshvar22 do you have opinions on that?